packages/e2e/step_definitions/login.step.ts

```diff
@@ -42,9 +42,13 @@ Then(/^I enter '(.*)' login credentials$/, async function(credentialsType: strin
   if (credentialsType === 'wrong') {
     await username.sendKeys(envConfig.INVALID_USER);
     await password.sendKeys(envConfig.INVALID_PASSWORD);
-  } else {
+  } else if (credentialsType === 'correct') {
     await username.sendKeys(TestRunContext.getTestUser().address);
     await password.sendKeys(envConfig.VALID_PASSWORD);
+  } else {
```

Comment:
What is meant by the 3rd case here, which is neither wrong nor correct?

Reply:
example from the registration feature:
` And I enter 'test3' login credentials`
test3 is the tag for the credentials I used when creating a new user via the registration UI.
Now I want to log in using those ones rather than the test user credentials created at the beginning of the tests